linera-core/src/unit_tests/worker_tests.rs

```diff
@@ -539,29 +540,29 @@ where
 
 #[test(tokio::test)]
 async fn test_memory_handle_block_proposal_with_chaining() {
-    let client = MemoryStoreClient::make_test_client(None).await;
+    let client = MemoryStoreClient::make_test_client(None, TestClock::new()).await;
```

Comment:
Isn't the test_client always using the `TestClock`?

Reply:
I didn't dare to do that for all tests yet, but maybe I should… I'll try it out.